docs: Mend hosted apps no longer using encrypted secrets (#35678)
renovatebot/renovate: docs/usage/mend-hosted/credentials.md

## :warning: Migrate secrets in your Renovate config file :warning:

> The Mend Renovate cloud apps no longer read encrypted secrets from Renovate config files in your repositories.

Reviewer: ```suggestion
The Mend Renovate cloud apps no longer read `encrypted` secrets from Renovate config files in your repositories.
```

> You must migrate any secrets you currently keep in a Renovate config file, and put them in the app settings page on [developer.mend.io](https://developer.mend.io).

Reviewer: ```suggestion
You must migrate any secrets you currently keep in a Renovate config file, and upload them as secrets to org or repo settings pages on [developer.mend.io](https://developer.mend.io).
```
